composer.json

```diff
@@ -52,5 +52,8 @@
             "php": "7.4.20"
         },
         "sort-packages": true
+    },
+    "require-dev": {
+        "phpstan/phpstan-symfony": "^1.0"
```

Comment:
Just a reminder for myself, it probably makes sense to move phpstan from tools to dev-dependencies as well, since it will be pulled by composer anyway.